frontend/express/public/javascripts/countly/vue/components/input.js

```diff
@@ -530,6 +531,14 @@
                     "cly-vue-select-x__pop--has-single-option": this.hasSingleOption
                 };
             },
+            showSelectedCount: function() {
```

Comment:
This could be simplified as explained here: https://stackoverflow.com/a/49208677